pom.xml

```diff
@@ -148,11 +142,11 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-validation</artifactId>
+            <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
```

Comment:
We don't have any webflux API being used in our code. Is this library mandatory for starter-web project? Could you try to remove and verify once.

Reply:
removed web-flux dependency and its building fine, I also tried commenting spring-boot-starter-web dependency and it is building fine should I remove that too ?